Week3/homework/2-step3.js

```diff
@@ -3,6 +3,27 @@
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const values = [];
   // Add your code here
+  for (let i = startIndex; i <= stopIndex; i++) {
+    values.push(i);
+  }
+
+  for (let i = 0; i <= values.length; i++) {
+    if (values[i] % 3 === 0 && values[i] % 5 === 0) {
```

Comment:
Hi @Wael-Alhomsi - I think you can also write the `if` block as `if (values[i] % 15 === 0)`. If a number is both divisible by 3 and 5, it would mean that it would always be divisible by 15. This way you can avoid the extra `&&` condition.

Reply:
Otherwise, you were able to ace the task. 😉